packages/typegpu/tests/builtin.test.ts

```diff
@@ -1,12 +1,12 @@
 import { describe, expect, expectTypeOf, it } from 'vitest';
-import * as d from '../src/data';
 import {
   type BuiltinPosition,
   type BuiltinVertexIndex,
   type OmitBuiltins,
-  StrictNameRegistry,
   builtin,
-} from '../src/experimental';
+} from '../src/builtin';
+import * as d from '../src/data';
```

Comment:
Let's use the builtin inside `d`. That way, we can ensure that builtins are properly exported from a public entry point.